packages/shared/src/telemetry/collector.ts

```diff
@@ -236,10 +236,8 @@ export class TelemetryCollector implements TelemetryCollectorInterface {
       version: this.#metadata.sdkVersion,
     };
 
-    // @ts-expect-error -- The global window.Clerk type is declared in clerk-js, but we can't rely on that here
```

Comment:
Instead of casting, we can probably define the type on window in this module. The Clerk type should be available in the types package